scripts/ci/after_success.sh

```diff
@@ -16,6 +16,16 @@ fi
 cd sentinel && npm install && cd .. && \
 cd api && npm install && cd ..
 
+function tagSubmodules {
+    cd api
+    git tag $TRAVIS_TAG
+    git push --tags
+    cd ../sentinel
+    git tag $TRAVIS_TAG
```

Comment:
Can we fix the duplication here? Either pull out another function, `tagSubmodule`, or loop over `['api', 'sentinel']` ?